src/cpp/shared/gui/msg.cpp

```diff
@@ -31,3 +35,13 @@ void GUIBroadcastMessage(heroWindow* hwnd, int f, int c, void* p) {
 	evt.payload = p;
 	hwnd->BroadcastMessage(evt);
 }
+
+int GUIGetDropdownSelection(heroWindow* hwnd, void* p) {
+  tag_message evt;
+  evt.yCoordOrFieldID = 100;
+  evt.eventCode = INPUT_GUI_MESSAGE_CODE;
+  evt.xCoordOrKeycode = 55;
```

Comment:
It was used in the original HoMM2 function and I am using it again here. I would have a defined constant here, but I'm afraid that one doesn't already exist and I don't think that I am currently able to identify what it should be defined as, if anything.